api/client/src/main/java/org/projectnessie/client/auth/oauth2/Secret.java

```diff
@@ -38,27 +31,11 @@ public int length() {
     return value.length;
```

Comment:
Nit: refactor `length()` to `isNotEmpty()` - and make the function package-private